src/shared/ExtensionMessage.ts

```diff
@@ -41,7 +41,11 @@ export interface ExtensionMessage {
 		| "userCreditsPayments"
 		| "totalTasksSize"
 		| "addToInput"
+		| "relativePathResponse" // Added for drag-and-drop file path response
```

Comment:
is it possible to remove `relativePathResponse` and `relativePath` and instead just use `relativePathsResponse`/`paths` with single elements in the array?